lib/galaxy/managers/groups.py

```diff
@@ -119,25 +112,17 @@ def _get_group(self, sa_session: galaxy_scoped_session, group_id: int) -> model.
             raise ObjectNotFound("Group with the provided id was not found.")
         return group
 
-    def _get_users_by_encoded_ids(
-        self, sa_session: galaxy_scoped_session, encoded_user_ids: List[EncodedDatabaseIdField]
+    def _get_users_by_ids(
+        self, sa_session: galaxy_scoped_session, user_ids: List[DecodedDatabaseIdField]
     ) -> List[model.User]:
-        user_ids = self._decode_ids(encoded_user_ids)
         return get_users_by_ids(sa_session, user_ids)
 
-    def _get_roles_by_encoded_ids(
-        self, sa_session: galaxy_scoped_session, encoded_role_ids: List[EncodedDatabaseIdField]
+    def _get_roles_by_ids(
+        self, sa_session: galaxy_scoped_session, role_ids: List[DecodedDatabaseIdField]
     ) -> List[model.Role]:
-        role_ids = self._decode_ids(encoded_role_ids)
         stmt = select(Role).where(Role.id.in_(role_ids))
         return sa_session.scalars(stmt).all()
```

Comment:
Similar to `get_users_by_ids` this function can be moved to `lib/galaxy/managers/roles.py`